src/ComposerRequireChecker/Exception/FileParseFailed.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ComposerRequireChecker\Exception;
+
+use RuntimeException;
+use Throwable;
+
+class FileParseFailed extends RuntimeException
+{
+  public function __construct(string $file, Throwable $previous = NULL)
+  {
+    parent::__construct($file . ': ' . $previous->getMessage(), 0, $previous);
```

Comment:
This error message should also state that the parsing failed.

For example, one of the original exceptions reported: `Parsing the file [%s] resulted in an error.` (much more descriptive).

Can we perhaps add some prefixing wording to the message here?

Reply:
How about `sprintf('FileParseFailed: Parsing the file [%s] resulted in an error: %s', $file, $previous->getMessage());`?

Reply:
I do believe the prefix might be redundant though, as the exception class is also printed when run in the console

Reply:
I think `Parsing the file [%s] resulted in an error: %s` should be enough 👍

Reply:
The suggested improvement works, although `FileParseFailed:` in front of it is indeed not needed :+1:

Reply:
Done! 👍🏻